### ISSUE TITLE
Discrepancy between ase eam and matscipy eam

### ISSUE THREAD

**Issue by gabor1** [libAtoms/matscipy#30]
[pot.fs.zip](https://github.com/libAtoms/matscipy/files/4232059/pot.fs.zip)

I am trying to use an EAM potential for FeH, and have a `.fs` file. When I pass it to the matscipy calculator, I get the same energy that the standard ASE EAM calculator gives, but the forces are different. Geometry optimisation fails with the matscipy calculator, so I that is the wrong one. Am I doing something wrong? 

```
import matscipy
import ase
from ase.lattice.cubic import BodyCenteredCubic
import numpy as np

calc1 = matscipy.calculators.eam.EAM("pot.fs", kind="eam/fs")   
calc2 = ase.calculators.eam.EAM(potential='pot.fs')  
a = BodyCenteredCubic(directions=[(1,0,0), (0,1,0), (0,0,1)], size=(3,3,3), symbol='Fe', pbc=True, latticeconstant=2.855)
a.append(ase.Atom('H', (0.0,a0*0.25,a0*0.5)))
a.set_calculator(calc1)
e1 = a.get_potential_energy()
f1 = a.get_forces()
a.set_calculator(calc2)
e2 = a.get_potential_energy()
f2 = a.get_forces()
print(e1-e2)
print(np.max(f1-f2))
```

comes back with

```
2.842170943040401e-14
0.028209307473157738
```

if I don't add the hydrogen, the forces are the same (even after a rattle).

**Comment by pastewka** [comment]
This sounds like a bug.

@wgnoehring did you notice any problems and can you have a look at this?

**Comment by pastewka** [comment]
Fixed with #34
